editor/js/Viewport.js

```diff
@@ -220,6 +226,14 @@ function Viewport( editor ) {
 
 	}
 
+	function isUIInteraction( event ) {
```

Comment:
Have you tried adding a invisible dom element to `VisualHelper` that listens for mouse events and has a `event.stopPropagation()`?

Reply:
Basically, to let the browser decide where the mouse event went instead of doing it yourself.

Reply:
> Have you tried adding a invisible dom element to VisualHelper that listens for mouse events and has a event.stopPropagation()?

No, not yet^^. Let me give this a try 👍 .

Reply:
Okay, I've updated the PR. Using an invisible DOM element means less changes to `Viewport`. Minor disadvantage. Orbiting does now not work anymore if you move with pressed mouse button over the helper. However, this is also true for the toolbar.

Reply:
Nice!